e2e-tests/operator-self-healing/run

```diff
@@ -7,6 +7,11 @@ test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 cluster="no-limits"
 
+if version_gt "1.20"; then
+	echo "Kubernetes version is 1.20+ so skipping this test because of pumba support."
```

Comment:
@tplavcic  why do we need to have pumba tests and chaos_mesh? Maybe we can use chaos_mesh only.

Reply:
I believe the pumba `pause` test still has sense, so I have asked to left it

Reply:
ok, LGTM 
@tplavcic  please update all needed jobs  in Percona-Lab/jenkins-pipelines repo too